internal/dataplane/parser/translators/ingress_atc.go

```diff
@@ -27,8 +27,11 @@ var (
 )
 
 var (
```

Comment:
nit: any reason why those can't be `const`?

Reply:
Used `const` here.